src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,97 @@
+package duke;
+
+import duke.task.Task;
+
+/**
+ * TaskList class handles the task list of Duke.
+ */
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructs the TaskList object.
+     *
+     * @param tasks Arraylist of tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Adds a task to the list of tasks.
+     *
+     * @param task Task to be added.
+     */
+    public void addTask(Task task) {
+        tasks.add(task);
+    }
+
+    /**
+     * Retrieves a task from the task list.
+     *
+     * @param i Index of task in task list.
+     * @return Task in task list.
+     */
+    public Task getTask(int i) {
+        return tasks.get(i);
+    }
+
+    /**
+     * Removes a task from the task list.
+     *
+     * @param i Index of task in task list.
+     * @return Task removed from task list.
+     */
+    public Task removeTask(int i) {
+        return tasks.remove(i);
+    }
+
+    /**
+     * Retrieves the task list.
+     *
+     * @return List of tasks.
+     */
+    public ArrayList<Task> getTasks() {
+        return tasks;
+    }
+
+    /**
+     * Returns the size of the task list.
+     *
+     * @return Size of task list.
+     */
+    public int getSize() {
+        return tasks.size();
+    }
+
+    /**
+     * Returns String representation of task list.
+     *
+     * @return String representation of task list.
+     */
+    public String getTaskList() {
+        String list = "These are the tasks in your list:\n";
+        for (int i = 0; i < tasks.size(); i++) {
+            list += (i + 1) + ". " + tasks.get(i).toString() + "\n";
```

Comment:
Should line breaks be used here to facilitate ease of reading?
Perhaps you can consider adding a break before the `+` operator in line with the [Java Coding Standard](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html#variables).